docs/kss/starter-pages/details-2.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<head>
+    <title>Techne: Layout Template - Details</title>
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="../public/dist/css/techne.min.css">
+</head>
+<style>
+    
+    .nav-tabs.nav-tabs--center{
+        margin-top: 30px;
+    }
+
+    .section-header{
+        margin-top: 25px;
+    }
+
+    .help-inline--on .hyicon-helpoff:before{
+        margin-top: 5px;
+    }
+
+</style>
+<body>
+
+    <!-- page header  -->
+    <div class="navbar navbar-default y-navbar-centered y-navbar-lg">
+        <div class="container-fluid">
+            <div class="navbar-left">
+                <div class="y-toolbar">
+                    <div class="y-toolbar-cell">
+                        <button class="btn-subordinate btn navbar-btn">
+                        <span class="hyicon hyicon-back"></span> <span class="hidden-xs">Back</span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div class="y-navbar-center">
+                <div class="y-navbar-center-content">
+                    <h1>Page Title</h1>
+                </div>
+            </div>
+            <div class="navbar-right">
+            </div>
+        </div>
+    </div>
+
+    <!-- tabs -->
+    <ul class="nav nav-tabs nav-tabs--center nav-tabs--responsive" role="tablist" id="demo-tabs">
```

Comment:
add a container around this element for the margin instead of adding the margin directly to the element